### PR TITLE
v0.1.8

### PR DESCRIPTION
**Pull request by andrejak**
Attempting to fix the docs:
- split up the content of the readme (it didn't really make sense to show the badges in our UI, and added a link to usage instructions in the one that'll be in the home of the repo on Github)
- maybe adding mdx parcel plugin to server will fix it...
- it all works fine on my machine, but may need to publish to really test...

### CODE REVIEW

File: packages/server/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "lint": "set -ex; tsc --build; tslint --project ."
   },
   "dependencies": {
+    "@mdx-js/parcel-plugin-mdx": "^1.0.18",
```
Comment (coffee-cup) [inline packages/server/package.json]: Remove this package from `packages/ui`

Comment (andrejak) [reply packages/server/package.json]: Tried, but it looks like it's needed for the Readme to work.

Comment (coffee-cup) [reply packages/server/package.json]: Hmm, then it probably doesn't have to be apart of the server... Can try and publish to see

Comment (andrejak) [reply packages/server/package.json]: Yeah, it didn't work :(

**Comment by coffee-cup** [comment]
Does the current version work?

**Comment by andrejak** [comment]
No, I guess I can take out the changes again until we figure it out (asked on mdx spectrum in case they had any ideas, no response yet)

**Comment by andrejak** [comment]
Or make a separate clean release branch and leave this open :/

File: README.mdx [hunk 2]
```diff
@@ -21,101 +21,7 @@
...
-
-## Examples
-
-[Here](https://github.com/prodo-ai/snoopy/tree/master/examples/01-counter) is a
-basic example of a Counter project set up for testing with Prodo. In it, we test
-the following components: `App`, `Button` and `Counter`.
-
-We also test Prodo with Prodo, so you can check [our source
-code](https://github.com/prodo-ai/snoopy/tree/master/packages/ui/src) directly.
+[Please read the usage instructions here.](packages/ui/README.mdx)
```
Comment (SamirTalwar) [inline README.mdx]: Why do we need to move this? Makes it way less accessible.

Comment (SamirTalwar) [reply README.mdx]: Can we use a symlink instead?

**Pull request merged**